client/my-sites/plans/current-plan/index.jsx

```diff
@@ -164,12 +164,17 @@ class CurrentPlan extends Component {
 		} = this.props;
 
 		const currentPlanSlug = selectedSite.plan.product_slug,
-			isLoading = this.isLoading();
-
-		const planConstObj = getPlan( currentPlanSlug ),
-			planFeaturesHeader = translate( '%(planName)s plan features', {
-				args: { planName: planConstObj.getTitle() },
-			} );
+			isLoading = this.isLoading(),
```

Comment:
I know it was like that before, but I believe the convention at A8C is to only declare one variable by statement.

Reply:
Oh good to know!  Awesome, will do. 👍